### ISSUE TITLE
[Feature Request] Adding an option to either display the Vanilla settings or the Sodium settings

### ISSUE THREAD

**Issue by Scarlaid** [LimeShulkerBox/better-sodium-video-settings#20]
Reading is confusing so I will try to explain as clear as possible

So normally, if this mod isn't around, in order to go back to the Vanilla settings, you have to execute `Shift + P`
And this mod will always show the Vanilla settings at launch

Any chance for an option to show either the Sodium settings panel or the Vanilla settings panel at launch?

**Comment by piemanau** [comment]
Do you mean like when you go to the video options it takes you to sodium instead with an option to go to the vanilla one?

**Comment by Scarlaid** [comment]
> Do you mean like when you go to the video options it takes you to sodium instead with an option to go to the vanilla one?

Yep, and there will be a button to go to the Vanilla video options, instead of `intended Shift+P like usual`

**Comment by piemanau** [comment]
Hmmm I will consider it but I think it kind of ruins the purpose of the mod
